apps/server/src/services/sheet-service/sheetUtils.ts

```diff
@@ -74,7 +74,6 @@ export function cellRequestFromEvent(
   worksheetId: number,
   metadata,
 ): sheets_v4.Schema$Request {
-  const returnRows: sheets_v4.Schema$CellData[] = [];
   const tmp = Object.entries(metadata)
```

Comment:
@alex-Arc can you help me find a better name for this? What is in this array?

Reply:
it could be rowData as it is just an array of cellData

Reply:
Makes sense. And what is the filtering doing?

Reply:
I think it takes out all the stuff that was not found in the sheet
so if there was no duration in the sheet it is removed here so it dose not become part of the upload
it is then sorted so it lands in the correct order